HPXMLtoOpenStudio/resources/waterheater.rb

```diff
@@ -1179,7 +1179,7 @@ def self.get_default_heating_capacity(fuel, num_beds, num_water_heaters, num_bat
       else
         cap_kw = 5.5
       end
-      return UnitConversions.convert(cap_kw, 'kW', 'kBtu/hr')
+      return UnitConversions.convert(cap_kw * 1000.0, 'W', 'Btu/hr')
```

Comment:
This correctly changes the unit conversions when the fuel is electricity, but other fuel types are still wrong.

Also, I would recommend keeping the method how it was, since other code is expecting the units it was returning, and instead accommodate the unit conversion in measure.rb.